### PR TITLE
Upgrade webui to Bootstrap 5

### PR DESCRIPTION
**Pull request by saurori**
### Description

This PR updates the webui from Bootstrap 3 to Bootstrap 5. The CSS and HTML was updated, but not the JS (only the Dropdown plugin is used).

Related Issue: https://github.com/contribsys/faktory/issues/359

### Notes
- I tried to fix and update all RTL CSS but there's a chance I missed things. I didn't touch `webui/static/application-rtl.css` at all.
- Fixed a bug when `TextDirection` was missing in locales yml it would set the `<html dir="TextDirection">`.
- I removed some HTML from the navbar related to `navbar-livereload` as it did not seem to be in use.
- I modified the Bootstrap 3 JS `data-` selector names so it lines up with CSS selectors in Bootstrap 5. Bootstrap 5 does not use jQuery but the Faktory UI still relies on it so it was easier to leave that in place vs including additional JS to support the new dropdown JS code.

### CODE REVIEW

**Comment by mperham** [comment]
The Scheduled page lists jobs which contain the `at` attribute and have not yet executed.

https://github.com/contribsys/faktory/wiki/The-Job-Payload#options

**Comment by saurori** [comment]
Ah ok. Tested that, the page looks fine.

File: webui/index.ego [hunk 1]
```diff
@@ -19,7 +19,7 @@ func ego_index(w io.Writer, req *http.Request) {
       <span class="dot"></span>
     </span>
   </h3>
-  <div class="interval-slider ltr">
```
Comment (mperham) [inline webui/index.ego:22]: I believe this was put here because the slider doesn't make sense when displayed as rtl.

File: webui/layout.ego [hunk 2]
```diff
@@ -17,9 +17,10 @@ func ego_layout(w io.Writer, req *http.Request, yield func()) {
 
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
 
-    <link href="<%= relative(req, "/static/bootstrap.css") %>" media="screen" rel="stylesheet" type="text/css" />
```
Comment (mperham) [inline webui/layout.ego:20]: Bootstrap is always required. rtl overrides some of the base classes.

**Comment by mperham** [comment]
Lots of nice work here, thanks.

I noticed you didn't touch application-rtl.css. Did you test RTL? I believe you can do so by selecting an RTL language on the /debug page if you didn't already.

I will walk through the pages later today to double check the changes.

File: webui/index.ego (hunk 1 above)
Comment (saurori) [reply webui/index.ego:22]: This is how it looks currently (I just modified english to RTL). If this is incorrect, I'll putt he `ltr` class back:

File: webui/layout.ego (hunk 2 above)
Comment (saurori) [reply webui/layout.ego:20]: Bootstrap 5 has it's own RTL, not a 3rd party CSS. The RTL CSS is a complete replacement, not an addition. See: https://getbootstrap.com/docs/5.0/getting-started/rtl/

**Comment by saurori** [comment]
I tested RTL a bunch, as much as I know how (native English speaker). It all looks good to me but there may be some subtleties I missed.

**Comment by saurori** [comment]
I did another pass over `application-rtl.css` and mostly removed dead CSS but also fixed a few small layout bugs / replaced items with native Bootstrap RTL classes. https://github.com/contribsys/faktory/pull/360/commits/dcca7ef63061e9d836180666f3eb9b98ad546d71

**Comment by mperham** [comment]
There are a few minor issues here and there. Some before and after images:

Before
before

After
after

Button color is gone. The table rows are colored differently, I don't mind that but do they need a little more contrast vs the page body color?

**Comment by mperham** [comment]
But overall, where things look different, they look **better** to my eye so I'm quite pleased!

I think the Dashboard interval slider needs to retain its `ltr` class. That class has been there for years and no one has complained so please restore it. I'm happy to remove it if someone with rtl experience tells me it looks wrong.

**Comment by saurori** [comment]
1. Buttons: BS5 got rid of `button-default`, will fix that
2. For whatever reason, BS5 switches the table row colors of `:odd` vs `:even`. Otherwise it's close to the same as before. I agree the grey color on tables could use more contrast. `table-light` looks a bit better, see [variants](https://getbootstrap.com/docs/5.0/content/tables/#variants) or a custom color...

File: webui/index.ego (hunk 1 above)
Comment (saurori) [reply webui/index.ego]: Fixed in: https://github.com/contribsys/faktory/pull/360/commits/6c1ab89251f1adf3e1d29b215cef90ae6a1c4883

**Comment by mperham** [comment]
Thanks, I'm ready to merge this. Are you happy with it or do you need more time?

I think you've earned commit bit with this work. You are welcome to make changes, just please open a PR for anything you do change so I can review and we have a record of why things changed.

**Comment by saurori** [comment]
👍 I think it's ready to merge.

**Comment by mperham** [comment]
Oh, and any significant change should have a line in Changes.md. Can you add one for this PR?

**Comment by saurori** [comment]
Changelog: https://github.com/contribsys/faktory/pull/363

**Pull request merged**